app/server/api/credit_transfer_api.py

```diff
@@ -132,7 +132,7 @@ def put(self, credit_transfer_id):
             }
             return make_response(jsonify(response_object)), 404
 
-        if credit_transfer.transfer_status.value != 'PENDING':
+        if credit_transfer.transfer_status.value not in ['PENDING', 'PARTIAL']:
             response_object = {
                 'message': 'Transfer status is {}. Must be PENDING to modify'
```

Comment:
Update response message to `Transfer status is {}. Must be PENDING or PARTIAL to modify`